src/sentry/static/sentry/app/stores/organizationEnvironmentsStore.jsx

```diff
@@ -23,11 +24,17 @@ const OrganizationEnvironmentsStore = Reflux.createStore({
   },
 
   loadInitialData(environments) {
-    this.items = environments.map(this.makeEnvironment);
+    this.environments = environments.map(this.makeEnvironment);
+    this.trigger({environments: this.environments, error: this.error});
+  },
+
+  onFetchOrganizationEnvironmentsError(error) {
+    this.error = error;
+    this.trigger({items: this.items, error});
```

Comment:
`items` doesn't exist

Reply:
woops my bad

Reply:
If there is an error do you want to clear out the previous environments?